### PR TITLE
chore: managed dimension sample

### PR DESCRIPTION
**Pull request by tpluscode**
Proposed API resources to hold dimension mappings

The CI will fail, to be fixed in subsequent PRs

### CODE REVIEW

File: fuseki/sample-ubd.trig [hunk 1]
```diff
@@ -82,13 +83,39 @@ graph <cube-project/ubd/dimensions-metadata> {
 
   <cube-project/ubd/dimensions-metadata/pollutant>
     schema:about <https://environment.ld.admin.ch/foen/ubd/28/pollutant> ;
+    scale:scaleOfMeasure scale:Nominal ;
+    cc:dimensionMapping <cube-project/ubd/dimension-mapping/pollutant> ;
```
Comment (tpluscode) [inline fuseki/sample-ubd.trig:87]: This would be created when a dimension is set to `Nominal`

File: fuseki/sample-ubd.trig [hunk 2]
```diff
@@ -110,7 +137,7 @@ graph <cube-project/ubd/cube-data> {
     <https://environment.ld.admin.ch/foen/ubd/28/aggregation>     <https://environment.ld.admin.ch/foen/ubd/28/aggregation/annualmean> ;
     <https://environment.ld.admin.ch/foen/ubd/28/dimension/value> "6.2"^^xsd:float ;
     <https://environment.ld.admin.ch/foen/ubd/28/dimension/year>  "2003"^^xsd:gYear ;
-    <https://environment.ld.admin.ch/foen/ubd/28/pollutant>       <https://environment.ld.admin.ch/foen/ubd/28/pollutant/so2> ;
+    <https://environment.ld.admin.ch/foen/ubd/28/pollutant>       "so2" ;
```
Comment (tpluscode) [inline fuseki/sample-ubd.trig:140]: Being an unmapped value, `"so2"` would appear as string coming form the CSV

File: fuseki/sample-ubd.trig [hunk 3]
```diff
@@ -132,7 +159,7 @@ graph <cube-project/ubd/cube-data> {
     <https://environment.ld.admin.ch/foen/ubd/28/aggregation>     <https://environment.ld.admin.ch/foen/ubd/28/aggregation/annualmean> ;
     <https://environment.ld.admin.ch/foen/ubd/28/dimension/value> "5.4"^^xsd:float ;
     <https://environment.ld.admin.ch/foen/ubd/28/dimension/year>  "1999"^^xsd:gYear ;
-    <https://environment.ld.admin.ch/foen/ubd/28/pollutant>       <https://environment.ld.admin.ch/foen/ubd/28/pollutant/so2> ;
+    <https://environment.ld.admin.ch/foen/ubd/28/pollutant>       <https://environment.ld.admin.ch/foen/ubd/28/pollutant/co> ;
```
Comment (tpluscode) [inline fuseki/sample-ubd.trig:162]: Changed this to `CO` to show that a value already mapped would be replaced by the correct URI

File: fuseki/sample-ubd.trig [hunk 4]
```diff
@@ -164,7 +191,10 @@ graph <cube-project/ubd/cube-data> {
     sh:path      cube:observedBy .
 
   _:pollutant
-    sh:in        ( <https://environment.ld.admin.ch/foen/ubd/28/pollutant/so2> ) ;
+    sh:in        (
+                   "so2" # <https://environment.ld.admin.ch/foen/ubd/28/pollutant/so2>
+                   <https://environment.ld.admin.ch/foen/ubd/28/pollutant/co>
+                 ) ;
```
Comment (tpluscode) [inline fuseki/sample-ubd.trig:197]: Finally, the Cube Shape would be a mix of mapped and unmapped value until all mappings are created.

File: fuseki/sample-ubd.trig (hunk 1 above)
Comment (martinmaillard) [reply fuseki/sample-ubd.trig:87]: You mean when a mapped managed dimension is selected right? Not all Nominal dimensions will be mapped

Comment (tpluscode) [reply fuseki/sample-ubd.trig:87]: > You mean when a mapped managed dimension is selected right?

Actually no. See that the `cc:managedDimension` is property of the mapping resource and not the metadata resource. But you should have the option to unselect it to clear the dictionary and "disable" the mapping

Comment (martinmaillard) [reply fuseki/sample-ubd.trig:87]: 👍

**Pull request merged**